x-pack/filebeat/input/http_endpoint/config.go

```diff
@@ -38,12 +40,14 @@ func defaultConfig() config {
 		URL:           "/",
 		Prefix:        "json",
 		ContentType:   "application/json",
+		SecretHeader:  "",
+		SecretValue:   "",
 	}
 }
 
 func (c *config) Validate() error {
 	if !json.Valid([]byte(c.ResponseBody)) {
-		return errors.New("response_body must be valid JSON")
+		return errors.New("Response_body must be valid JSON")
```

Comment:
The setting in the config file is named `respone_body`. Let's keep it lower case.